README.md

```diff
@@ -1 +1,82 @@
-# MyEnumerables
\ No newline at end of file
+<a name="readme-top"></a>
+
+## 📗 Table of Contents
+
+- [My Enumerables ](#my-enumerables-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [💖 Show your support ](#-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# My Enumerables <a name="about-project"></a>
+
+**My Enumerables** is a Ruby programme that loops through any given argument and outputs the result of the arguments.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- Ruby
+
+### Key Features <a name="key-features"></a>
+
+- Loop through all given lists
+- Loop Through any given list
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+- -👤 **Felix Sowah**
+- -👤 **Uchechi Nwaka**
+
+- -- GitHub: [@uchexm](https://github.com/uchexm)
+
+  -- LinkedIn: [@nwakauc](https://www.linkedin.com/in/nwakauc/)
+
+  -- Twiter [@nwakauc](https://twitter.com/Nwakauc)
+
+  👤 **Felix Sowah**
+
+- GitHub: [@githubhandle](https://github.com/Lancelot-SO)
+- Twitter: [@twitterhandle](https://twitter.com/Lancelot_hans)
+- LinkedIn: [LinkedIn](hhtps://www.linkedin.com/in/felix-sowah)
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Sort by date**
+- [ ] **Map by parameters**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💖 Show your support <a name="support"></a>
+
+If you like this project, please consider giving it a ⭐.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all code reviewers for making this project better.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```

Comment:
- [ ] Kindly add the MIT license for your project this will make your readme more professional, you can do it here on [Github](
https://github.com/uchexm/MyEnumerables/community/license/newbranch=dev&filename=LICENSE&template=mit)

Reply:
Aye Aye Captain 👌